frontend/src/components/ui-kit/components/Button/Button.module.scss

```diff
@@ -25,11 +30,23 @@
     background-color: $green-dark-color;
   }
 }
+.btn-catalogue {
+  padding: 12px 25px;
+}
+.btn-add {
+  padding: 12px 12px 12px 25px;
+}
+.btn-added {
```

Comment:
I don't like these names, they are tied to a specific button, but the content of the button can be different